internal/rest/dequeue_nodes.go

```diff
@@ -699,7 +699,15 @@ func (rest *RestOperations) ExecuteRestAndPopulateCache(rest_ops []*utils.RestOp
 
 func updateGatewayStatusWithVsError(key string, rest_op *utils.RestOp) {
 	if rest_op.Model == "VirtualService" {
-		vs := rest_op.Obj.(avimodels.VirtualService)
+		var vs avimodels.VirtualService
+		switch rest_op.Obj.(type) {
+		case avimodels.VirtualService:
+			vs = rest_op.Obj.(avimodels.VirtualService)
```

Comment:
Please describe in which case object type will be *avimodels.VirtualService and avimodels.VirtualService ?

Reply:
in case of child vs it will be *avimodels.VirtualService while in parent vs it is avimodels.VirtualService